.circleci/make-rpm-packages.sh

```diff
@@ -13,13 +13,9 @@ elif [ -z "$PARAM_DIST" ]; then
     exit 1
 fi
 
-yum install -y epel-release
 
 if [ "$PARAM_DIST" = centos ]; then
-  if [ "$PARAM_RELEASE" = 8 ]; then
-      dnf install -y 'dnf-command(config-manager)'
-      yum config-manager --set-enabled powertools
-  fi
+    yum install -y epel-release
 fi
```

Comment:
This adjustment should happen before fedora's introduction, modulus el8 removal.